data/timesketch.conf

```diff
@@ -33,6 +33,7 @@ OPENSEARCH_PASSWORD = None
 OPENSEARCH_SSL = False
 OPENSEARCH_VERIFY_CERTS = True
 OPENSEARCH_TIMEOUT = 10
+OPENSEARCH_FLUSH_INTERVAL = 5000
```

Comment:
I'm curious, is 5k better as a default for most cluster sizes?

Reply:
The plaso task uses 5000 as default and the run_csv_jsonl 1000. I have tested this out on the dev instance without any problems. But it can be adjusted to 1000, what do you think?